src/course-unit/sidebar/index.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import classNames from 'classnames';
+import { Card, useToggle } from '@openedx/paragon';
+import { InfoOutline as InfoOutlineIcon } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import ModalNotification from '../../generic/modal-notification';
+import { editCourseUnitVisibilityAndData } from '../data/thunk';
+import { getCourseUnitData } from '../data/selectors';
+import { PUBLISH_TYPES } from '../constants';
+import { SidebarBody, SidebarFooter, SidebarHeader } from './components';
+import useCourseUnitData from './hooks';
+import messages from './messages';
+
+const Sidebar = ({ blockId, isDisplayUnitLocation, ...props }) => {
```

Comment:
@PKulkoRaccoonGang I'm working on a [sidebar for tags](https://github.com/openedx/frontend-app-course-authoring/pull/852) that uses your sidebar as a base. [A discussion has arisen](https://github.com/openedx/frontend-app-course-authoring/pull/852#discussion_r1504518503) about how to make the sidebar styles reusable, perhaps that the sidebar component is a [wrapper for its content](https://github.com/openedx/frontend-app-course-authoring/pull/852#discussion_r1511780367) (until now we would have three sidebars). What do you think?

Reply:
I think this is a good idea 👍 
Do you plan to split the sidebar presented in this PR into a wrapper and sidebar content?